salt/metalk8s/addons/addons-service-config/configmap.sls

```diff
@@ -0,0 +1,41 @@
+#! metalk8s_kubernetes
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metalk8s-monitoring
+  name: metalk8s-grafana-config
+data:
+  config.yaml: |-
+    apiVersion: metalk8s.scality.com/v1alpha1
```

Comment:
We may want to give these their own group. WDYT @gdemonet

Reply:
`addons.metalk8s.scality.com`? Or even more specialized, like `<addon-name>.addons.metalk8s.scality.com`?